configuration/pipeline_config.json

```diff
@@ -0,0 +1,56 @@
+{
+  "PipelineName": "Test-Pipeline-Daniel",
+  "RawDataSources": [
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/experimental-test-text-it-token.txt",
```

Comment:
```suggestion
      "TokenFileURL": "gs://avf-credentials/experimental-test-textit-token.txt",
```

Reply:
rename gs file too for consistency with new pipeline names

Reply:
updated thanks